utils/appliance/__init__.py

```diff
@@ -1604,7 +1604,7 @@ def start_evm_service(self, log_callback=None):
         self._evm_service_command('start', expected_exit_code=0, log_callback=log_callback)
 
     @logger_wrap("Restart EVM Service: {}")
-    def restart_evm_service(self, rude=False, log_callback=None):
+    def restart_evm_service(self, rude=False, wait_for_web_ui=False, log_callback=None):
```

Comment:
This is very dangerous. You have changed the default behaviour of the ```restart_evm_service```. Of you had left it with True as the default and then altered all your calls to it to use ```False``` you would have been OK, but doing it this way requires you to go through every other usage in the codebase and switch them to ensure True is passed. I believe the set_yaml_config functions do not require a server restart, but I could be wrong.